tox.ini

```diff
@@ -15,3 +16,5 @@ commands =
   flake8 xsnippet_api/ tests/
 # check that unittests are passed
   py.test --cov xsnippet_api/ --cov-append tests/
+# check the swagger definition of the API
+  swagger-flex -s contrib/swagger.yaml
```

Comment:
> It's still a tox virtual environment, i.e. the xsnippet_api package will be installed before running the commands section from tox.ini

We can skip installation for `swagger`, since we don't need it. Example:

```
[testenv:swagger]
skip_install = true
deps =
  swagger-flex
commands =
  swagger-flex -s contrib/swagger.yaml
```

> I'm fine with moving swagger-flex into a separate testvenv and even support it. It's just I'll have to specify the basepython = python3.5 for that env.

I didn't get that. For what reason we need to use `basepython = python3.5`? In case of `tox` is run under `python2`?